Lib9c/TableData/Sheet.cs

```diff
@@ -3,11 +3,15 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+#if UNITY_EDITOR
 using System.Text;
+#endif
 using Bencodex.Types;
 using JetBrains.Annotations;
 using Nekoyume.Model.State;
+#if UNITY_EDITOR
 using Serilog;
+#endif
```

Comment:
is it correct changes?

Reply:
Yes, this is the right change.
These two using statements are used below with the same preprocessor.